zeth_contracts/contracts/Pghr13Mixer.sol

```diff
@@ -230,48 +225,39 @@ contract Pghr13Mixer is BaseMixer {
     }
 
     function verifyTx(
-        uint256[2] memory a,
-        uint256[2] memory a_p,
-        uint256[4] memory b,
-        uint256[2] memory b_p,
-        uint256[2] memory c,
-        uint256[2] memory c_p,
-        uint256[2] memory h,
-        uint256[2] memory k,
+        uint256[18] memory proof_data,
```

Comment:
Name is inconsistent with argument name in `verifyTx` for Groth16

Reply:
Renamed the groth16 version from `proof` to `proof_data`. (In the pghr13 case this function fills out an actual Proof structure, which is naturally named `proof`.  The groth16 version is optimized to pull directy from the memory buffer.)